.github/workflows/test.yaml

```diff
@@ -60,7 +60,8 @@ jobs:
 
         steps:
         - uses: actions/checkout@v2
-          fetch-depth: 0
+          with:
+            fetch-depth: 0
```

Comment:
I am not 100% sure why this fixes it.
What this does is cause the tags to be fetched, so this means the version is correct when it gets installed.

🤷🏻‍♀️ 

Probably should be doing this anyway everywhere!